src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/OperatorOverloadSyntaxClassifier.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Classification.Classifiers;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.Classification
+{
+    internal class OperatorOverloadSyntaxClassifier : AbstractSyntaxClassifier
+    {
+        public override void AddClassifications(
+            Workspace workspace,
+            SyntaxNode syntax,
+            SemanticModel semanticModel,
+            ArrayBuilder<ClassifiedSpan> result,
+            CancellationToken cancellationToken)
+        {
+            var symbolInfo = semanticModel.GetSymbolInfo(syntax, cancellationToken);
+            if (symbolInfo.Symbol is IMethodSymbol methodSymbol
+                && methodSymbol.MethodKind == MethodKind.UserDefinedOperator)
+            {
+                var operatorSpan = GetOperatorTokenSpan(syntax);
+                if (!operatorSpan.IsEmpty)
+                {
+                    result.Add(new ClassifiedSpan(operatorSpan, ClassificationTypeNames.OperatorOverload));
+                }
+            }
+        }
+
+        private static Text.TextSpan GetOperatorTokenSpan(SyntaxNode syntax)
+        {
+            if (syntax is BinaryExpressionSyntax binaryExpression)
+            {
+                return binaryExpression.OperatorToken.Span;
+            }
+            else if (syntax is PrefixUnaryExpressionSyntax prefixUnaryExpression)
+            {
+                return prefixUnaryExpression.OperatorToken.Span;
+            }
+            else if (syntax is PostfixUnaryExpressionSyntax postfixUnaryExpression)
+            {
+                return postfixUnaryExpression.OperatorToken.Span;
+            }
```

Comment:
do we care about assignment operators when the operator is overloaded?  i.e. a call to += when + is overloaded.

Reply:
Good catch. The fix for C# is very straightforward. For VB.NET it is a little more tricky since SemanticModel doesn't return SymbolInfo for AssignmentStatement. This probably explains why we don't show quick info for compound assignment in VB.NET.

Reply:
Interesting (and surprising about VB)!.  I would do the C# side, and then open a bug about the VB side and what we want to do about that.  It may be on you then to make VB work here in the compiler if you're interested in making that happen! :)

Reply:
https://github.com/dotnet/roslyn/issues/31678

Reply:
It's a long standing problem. The original bug is https://github.com/dotnet/roslyn/issues/6366.